package/MDAnalysis/core/topologyobjects.py

```diff
@@ -238,10 +234,13 @@ def angle(self):
         .. versionadded:: 0.9.0
         .. versionchanged:: 0.17.0
            Fixed angles close to 180 giving NaN
+        .. versionchanged:: 0.18.1
+           Added pbc keyword
```

Comment:
This would be worth a note in the changelog, I think.

Reply:
(Thanks for adding the CHANGELOG entry.)